bridges/snowbridge/primitives/inbound-queue/src/v2/traits.rs

```diff
@@ -21,3 +22,39 @@ pub enum ConvertMessageError {
 	/// Invalid network specified (not from Ethereum)
 	InvalidNetwork,
 }
+
+pub trait MessageProcessor<AccountId> {
+	/// Lightweight function to check if this processor can handle the message
+	fn can_process_message(who: &AccountId, message: &Message) -> bool;
```

Comment:
We know `who` is a message relayer, so we can just call it that, within this context. Is more descriptive

```suggestion
	fn can_process_message(relayer: &AccountId, message: &Message) -> bool;
```